cnf-tests/testsuites/e2esuite/ptp/ptp.go

```diff
@@ -31,10 +32,26 @@ const (
 	ptpDaemonsetName         = "linuxptp-daemon"
 )
 
+var (
+	isSingleNode bool
+)
+
+func init() {
+	singleNodeCluster := os.Getenv("SINGLE_NODE_CLUSTER")
+	if singleNodeCluster != "" {
+		isSingleNode, _ = strconv.ParseBool(singleNodeCluster)
+	} else {
+		isSingleNode, _ = nodes.IsSingleNode()
+	}
+}
+
 var _ = Describe("ptp", func() {
 
 	execute.BeforeAll(func() {
 		if !discovery.Enabled() {
+			if isSingleNode {
+				Skip("Running in single node mode")
```

Comment:
This is gonna go to users, I'd expand the reason a bit.
Something along the line of  "single node ptp test rely on external grandmasters, please run in discovery mode"